src/components/nextRegionModal.html

```diff
@@ -15,7 +15,7 @@ <h5 class="modal-title" data-bind="text: `Travel to ${GameConstants.camelCaseToS
                    <strong>There are a few things you should know before you depart:</strong><br/>
                    You will gain <strong data-bind="text: App.game.breeding.queueSlotsGainedFromRegion(player.highestRegion())"></strong> queue slots in the Hatchery.<br/>
                    <!-- ko if: App.game.challenges.list.regionalAttackDebuff.active() -->
-                   When Pokémon are in a Region they're not native to, they will only retain <strong data-bind="text: App.game.party.getRegionAttackMultiplier(player.highestRegion() + 1).toLocaleString(undefined,{style: 'percent'})"></strong> of their total attack.<br/>
+                   When Pokémon are in a Region they're not native to, they will only retain <strong data-bind="text: Party.getRegionAttackMultiplier(player.highestRegion() + 1).toLocaleString(undefined,{style: 'percent'})"></strong> of their total attack.<br/>
```

Comment:
I don't like this refactor. Party should not have any static methods. Either use `App.game.party` or move it to a helper class

Reply:
I have moved this code into more appropriate places and removed all static methods from Party